awscrt/s3.py

```diff
@@ -18,6 +18,51 @@
 from typing import Optional
 from enum import IntEnum
 
+class InstanceLock(NativeResource):
```

Comment:
Name it consistently with the C code
```suggestion
class CrossProcessLock(NativeResource):
```